content/kubernetes/release-notes/6-2-18-releases/k8s-6-2-18-41-jan-2023.md

```diff
@@ -16,6 +16,11 @@ This is a maintenance release of Redis Enterprise for Kubernetes 6.2.18-41 that
 
 New images and fixes are listed below. Refer to [6.2.18-41 (Dec 2022]({{<relref "/kubernetes/release-notes/6-2-18-releases/k8s-6-2-18-41.md">}}) for compatibility notes and known limitations.
 
+{{<warning>}} Due to a change in the SCC, on OpenShift clusters running version 6.2.12 or earlier upgrading to version 6.2.18 or later, where `node:1` is <b>not</b> the master node, the upgrade might get stuck.
+
+For more info and steps to prevent this issue, see [upgrade a Redis Enterprise cluster (REC)]({{<relref "/kubernetes/re-clusters/upgrade-redis-cluster.md#before-upgrading">}}).
```

Comment:
```suggestion
For more info and steps to prevent this issue, see [upgrade a Redis Enterprise cluster (REC)]({{<relref "/kubernetes/re-clusters/upgrade-redis-cluster#before-upgrading">}}).
```